packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -506,23 +403,6 @@ describe('Launchpad: Setup Project', () => {
         cy.contains('Project Setup')
       })
 
-      it('can reconfigure config after e2e has been set up', () => {
-        scaffoldAndOpenProject('pristine-with-e2e-testing')
-        cy.withCtx((ctx) => {
-          ctx.coreData.forceReconfigureProject = {
```

Comment:
We don't support "reconfigure" anymore... I checked the code base, we can remove all code related to `forceReconfigureProject`. I didn't want to bundle that refactor here, though.